### PR TITLE
Syndicate Black Friday - Sales all around! (Uplink Price rebalance)

### PR DESCRIPTION
**Pull request by Manly-Dwarf**
:cl: KawaiiBigBoss
balance: Altered the price of several traitor items.
tweak: Mindslave Implant is now 6 TC (from 8)
tweak: Greater Mindslave implant is now 10 TC (from 16)
tweak: Energy Chainsaw is now 14 TC (from 16)
tweak: Removed role restrictions on the energy chainsaw.
tweak: Energy Butterfly Knife, or Balisong, is now 8 TC (from 12)
tweak: Autosurgeon is now 1 TC (from 4)
tweak: Executioner's Sword is now 1 TC (from 3)
tweak: Gloves of the North Star is now 8 TC (from 10)
tweak: Syndicate Tome is now 2 TC (from 5, 9 on /tg/ branch)
tweak: Gremlin Grenade, Beenade, and Catnade have all had their prices shaved by 2 (2, 2, and 3 TC from 4, 4, and 5 TC respectively)
tweak: Powerfist is now 6 TC (from 8)
tweak: Codespeak Manual is now 0 TC (from 2 TC) but can only be purchased 4 times because Bawb got scared
tweak: Deluxe Codespeak Manual is now 3 TC (from 4 TC)
tweak: Altered the descriptions of the Codespeak and Deluxe Codespeak manuals
tweak: Martial Arts Scroll, or Sleeping Carp Scroll, is now 12 TC (from 17)
tweak: Box of Throwing Weapons is now 2 TC (from 3)
tweak: Dart Pistol is now 3 TC (from 4)
tweak: Syndicate Hardsuit, or is now 7 TC (from 8)
tweak: Syndicate Surgery Bag is now 1 TC (from 3)
tweak: Ancient Jumpsuit is now 0 TC (from 20) but can only be purchased 4 times because Bawb got scared
tweak: Reverse Revolver is now 13 TC (from 14)
/:cl:

Changes:
Mindslave Implant, 8 to 6 TC.
Reason: Nobody ever uses these, thanks to their time limit. The best you could do is create a grey tide, but with only one person your options are limited.

Greater Mindslave Implant: 16 to 10 TC.
Reason: Even less people use these. Blood Brother is this item but for free.

Energy Chainsaw: Available for all crew. 16 to 14 TC.
Reason: Is rarely seen due to the role restrictions, and doesn't really have much of a reason to be restricted after Chainsaws became craftable.
Is more or less a slightly worse version of the pre-rebase traitor chainsaw, and there are generally better options for mass murder such as the Lawnmower that the Botanists already get, or the Double Esword. 14 TC was the original price of ye olde chainsaw any how.

Energy Butterfly: 12 to 8 TC.
Reason: Inferior esword with a good gimmick and armor piercing. Has no reason to be as expensive as an ebow.

Autosurgeon: 4 to 1 TC.
Reason: You can already do surgery on yourself, the CMO already spawns with an autosurgeon, and the Nuke Ops do too. Come on!

Execution Sword: 3 to 1 TC.
Reason: Willing to raise to 2 TC, but for now this item kinda sucks. As strong as a fire extinguisher with a slow hit rate, but an incredibly funny gimmick. Not as good as an Energy Dagger nor as easy to conceal.

Gloves of the North Star: 10 to 8 TC.
Reason: Another gimmicky item. Fun, and the RNG stunlocks are great, but lacking compared to other items.

Syndicate Tome: 9 to 2 TC.
Reason: A bible that can also deal the damage of an edagger, and without the ability to conceal, and it reveals that you're a traitor to anyone who reads it?
Garbage!

All of the summon grenades: X to 2 or 3 TC.
Reason: The Beenade, Gremlin grenade, and Cat grenade are more of a nuisance or a gimmick than anything of real use. They usually get gunned down before they kill anyone.
Feral Cat grenade is a bit more expensive at 3 TC, but the rest are at 2.

Powerfist: 8 to 6 TC.
Reason: Got hit by the /tg/ grudgecode train. Isn't nearly as good as it was before.

Binary Encryption Key: 5 to 2 TC.
Reason: You can just talk to your subverted AI over the PDA.

Codespeak Manual: 2 to 0 TC. Limited Stock: 4.
Reason: Makes you an obvious traitor. Absolutely worthless, honestly. Bawb got scared and wanted a limited stock if it was 0 TC, so here you go.

Deluxe Codespeak Manual: 4 to 3 TC.
Reason: Makes you look like an obvious traitor. Being able to have a club of obvious traitors does not change this.

Martial Arts Scroll: 17 to 12 TC.
Reason: This is probably a controversial change, but the thing has seen some use and actually kind of sucks. It's good if you can stun someone, however its odd functions weigh it down. On top of that, not being able to use ranged weaponry and being melee only is suicide against Security, even if you can block their taser shots.

Box of Throwing Weapons: 3 to 2 TC.
Reason: Shurikens are strong, but the bola can already be crafted on their own and aren't as good as something like the DRAGNet.

Dart Pistol: 4 to 3 TC.
Reason: You can get a syringe gun literally anywhere else. But due to the existence of the chemical kit, this shouldn't have its price dropped too much.

Syndicate Hardsuit: 8 to 7 TC.
Reason: Nuke Ops get this for free, and while armor is great it won't save you from stuns or cuffs. Still a very solid item.

Syndicate Surgery Bag: 3 to 1 TC.
Reason: Due to the Autosurgeon nerf, this is pretty much necessary. On top of that it gives you a bunch of items you can already easily get on your own.

Ancient Jumpsuit: 20 to 0 TC. Limited Stock: 4.
Reason: Joke item, it's just a grey jumpsuit.

Reverse Revolver: 14 to 13 TC.
Reason: Joke item, easy to spot and can't be used if the Clown relinquishes his dignity and takes mutadone. Besides that, it's the only gun the Clown can reliably fire outside of the pie cannon.

"Why are you lowering all the costs of these items? SS13 is a game, you're not supposed to win."
I'm not lowering the price of things like the esword for a reason. Too many gimmicks right now are simply too expensive, and rely on the players you fight against to be bad in order to really succeed. Doing this will allow more people to do their gimmicks easier with more traitor items.
Also, a lot of these items simply suck. They have no reason to be as expensive as they are.

Willing to alter the prices of other items or raise/lower some others.

### CODE REVIEW

**Comment by KayeArray** [comment]
+1

**Comment by ghost** [comment]
KBB was working on this asking for my input on DMs
I'm pretty much good with all the proposed changes +1

**Review by KayeArray — DISMISSED** [review]

**Comment by tortellinitony** [comment]
no keep the ancient jumpsuit at 20

**Comment by Manly-Dwarf** [comment]
@tortellinitony 
It's literally just a grey jumpsuit. Assistants already spawn with one.
It does nothing. Has no reason to be 20 TC, even if it's a joke.

**Comment by praisenarsie** [comment]
lol this is shitty balance.

changing the uplink price of an item won't fix all of its problems. i learned this in a pr i made - https://github.com/tgstation/tgstation/pull/30287

all you're doing is literally giving people more options and practically free shit, disgusting.

word of advice, change the **functionality of the item** and not the price.

**Comment by Manly-Dwarf** [comment]
"I learned this in a PR I made"
"emag increased from 6 TC to 10 TC"
Uhh, no. Anyone could tell you that's a terrible idea, and they clearly did.

A lot of items don't need additional functionality. They're just overpriced. Making everything an overly gimmicky mess to "change" them is how you end up with items like the Power Fist.

also fuck off /tg/ shill.

**Comment by earthmoonjune** [comment]
+1

**Comment by GuyonBroadway** [comment]
Now making it so you can buy sleeping carp AND gloves of the north star is a pretty strong combo.  You gotta remember sleeping carp makes you do 11-17 damage per punch AND it ignores armour.

**Comment by Manly-Dwarf** [comment]
That is true, but you gotta remember that there's RNG shields and stunbatons which will instantly fuck you over, no questions asked. You also have to remember that the combo costs 20 TC, which is more than the Chainsaw and the Double Esword - which are both extremely powerful melee weapons in their own regard and still let you use ranged weaponry.
Melee only should be pretty powerful, especially if you're willing to spend 14-20 TC on melee only items.

**Comment by KayeArray** [comment]
100 kisses for making my spy gimmick more viable

**Comment by Manly-Dwarf** [comment]
I removed some of the spammy code, the costs are now all that's listed instead of the entire item, for most things.

**Comment by ghost** [comment]
+1

**Comment by JohnMcLaime** [comment]
Martial arts for 12 tc is too good, with gloves for 8 tc. Syndie surgery duffel bag gives you storage of duffel bag with no slowdown, i think, also there is that chest storage vest for 1 tc which is just a box which can be stored in beltslot, maybe buff it, make it 2 tc, same as surgery duff bag, but give it storage of trashbag? I mean you can already get trashbag in maint or steal from autistic janitor for free and it doesnt make you valido.

**Comment by KayeArray** [comment]
does role restricted answer your question

**Comment by JohnMcLaime** [comment]
What about rules regarding ancient jumpsuits/syndie language tomes? Since they are 0 tc, even with limited stock, why shouldnt you spam them around in some hallway? Would people, who wear/have them, 
be valid?

**Comment by JohnMcLaime** [comment]
Also, what about combat stims implant? It is still 8 tc after it was reworked and had it's anti stun stripped.

**Comment by Manly-Dwarf** [comment]
I don't know about the combat stims implant, all things considered.
As for Codespeak or Ancient Jumpsuits, it'd probably be just like picking up syndiesoap. Probably not a good idea to carry around, but also not really enough to get you instantly lynched on its own.
Also, ancient grey is restricted to Assistants, and makes them say "GLORF!" whenever they get hit.
The Martial Arts contract kinda sucks. Its main upside is the fact that you can't be hit by ranged attacks but that just means you get screwed over by stunbatons, flashbangs and Riot Shields. The Double Energy Sword is almost the exact same thing and kills people faster without a weird combat dial.
Sure, it combined with the Gloves of the North Star would be pretty powerful, but that's a 20 TC combo. Of course it should be powerful.
Lastly, the Combat Stims implant I've never seen anyone actually use, isn't it just an injection of omnizine at this point?

**Comment by JohnMcLaime** [comment]
The thing about martial arts is, that people dont expect you to have it so they will pull the gun out, while you can rush them with baton or something.
Combat stims got some chem which kinda heals you fast, but not as fast as hearty punch, the main reason to buy them is that you can activate them while in crit/stun, i am not sure if you can activate dentals/sec chem implant in crit, i think you can, just not while stunned.

**Comment by Manly-Dwarf** [comment]
That's fair, but in general you could already do that with any item + stunbaton.
The Gloves of the North Star are revealed when you examine someone, so if anyone is quick they can see that you have the gloves and you're traitor if yu're doing the combo.

**Comment by GuyonBroadway** [comment]
I need to buff the combat stims implant and reduce its cost.

File: hippiestation/code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -274,4 +264,44 @@
...
+/datum/uplink_item/device_tools/binary
+	cost = 2
+
+/datum/uplink_item/device_tools/codespeak_manual
+	desc = "Syndicate agents can be trained to use a series of codewords to convey complex information, which makes you look like an obvious traitor to anyone listening. This manual teaches you this Codespeak. You can also hit someone else with the manual in order to teach them. One use."
+	cost = 0
+	limited_stock = 4
+
+/datum/uplink_item/device_tools/codespeak_manual_deluxe
+	desc = "Syndicate agents can be trained to use a series of codewords to convey complex information, which makes you look like an obvious traitor to anyone listening. This manual teaches you this Codespeak. You can also hit someone else with the manual in order to teach them. This is the deluxe edition, which has unlimited uses. Now you and your club can get lynched together!"
```
Comment (Carbonhell) [inline hippiestation/code/modules/uplink/uplink_item.dm:281]: why did you leave the desc here?is it different than the original one?

File: hippiestation/code/modules/uplink/uplink_item.dm [hunk 2]
```diff
@@ -274,4 +264,44 @@
...
+	cost = 8
+
+/datum/uplink_item/device_tools/syndietome
+	cost = 2
+
+/datum/uplink_item/device_tools/binary
+	cost = 2
+
+/datum/uplink_item/device_tools/codespeak_manual
+	desc = "Syndicate agents can be trained to use a series of codewords to convey complex information, which makes you look like an obvious traitor to anyone listening. This manual teaches you this Codespeak. You can also hit someone else with the manual in order to teach them. One use."
```
Comment (Carbonhell) [inline hippiestation/code/modules/uplink/uplink_item.dm:276]: v

File: hippiestation/code/modules/uplink/uplink_item.dm (hunk 1 above)
Comment (Manly-Dwarf) [reply hippiestation/code/modules/uplink/uplink_item.dm:281]: Yes

File: hippiestation/code/modules/uplink/uplink_item.dm (hunk 2 above)
Comment (Manly-Dwarf) [reply hippiestation/code/modules/uplink/uplink_item.dm:276]: Same as above.

**Comment by Carbonhell** [comment]
code es oke

**Comment by Manly-Dwarf** [comment]
merge dis

**Comment by GuyonBroadway** [comment]
only has +2

**Comment by Manly-Dwarf** [comment]
more like +4

**Comment by GuyonBroadway** [comment]
tfw and flux are only GA's their +1's do not count unless in case of a tiebreaker, then their +1 counts as 1/3rd of a +1

**Comment by JohnMcLaime** [comment]
What does deluxe syndspeak manual do?

**Comment by Manly-Dwarf** [comment]
Jack shit.
It gives you a Codespeak Manual with infinite uses, meaning you can use it on yourself and your cronies.
As if anyone would actually use Codespeak.

**Comment by JohnMcLaime** [comment]
Doesnt tg have a tool to view how many times items were bought?

**Comment by Manly-Dwarf** [comment]
Not really, I don't think? You can restrict purchases simply by adding limited_stock = (number), but outside of that I'm not sure.

**Comment by praisenarsie** [comment]
they technically do. it's part of statbus (which is not endorsed by tg)

@atlanta-ned i summon thee

**Comment by praisenarsie** [comment]
fuck i tried

**Comment by feotusgod** [comment]
:boom: :camel: :fire:  make pda bombs 2 tc's, I always liked the idea that whenever pda bombing happened, people assume its nuke ops, but a regular atang could do it as well for the keks. I used to remember robust people disabling their pda messaging, or just throwing away pda's as superstition, but now absolutely nobody uses pda bombs... even the nuke ops... I guess people were allured by the fact that they were so cheap you could get a good chunk of the population when we used to have a high pop, but now its a waste of tc's...

**Comment by GuyonBroadway** [comment]
Also please list each uplink pricing rebalance in the changelog. Most players dont read the guthub so they might not know.

**Comment by Manly-Dwarf** [comment]
I was never told how to changelog, send help.

**Comment by GuyonBroadway** [comment]
In between the :cl:  and /:cl:  just put  

tweak: x now costs y telecrystals

**Comment by GuyonBroadway** [comment]
And do so for each item.

**Comment by Manly-Dwarf** [comment]
Will do. Thanks.

Updated the description.

**Comment by ghost** [comment]
+1

**Comment by tortellinitony** [comment]
@Kawaii-Big-Boss please make the syndicate balloon cost nothing as well

**Review by KayeArray — APPROVED** [review]
cuckonmycuckway

**Pull request merged**